### PR TITLE
tests: check `InContainer()` for `TestCPUValue` and `TestGetCgroupCPU()` test cases

### PR DESCRIPTION
**Pull request by Defined2014**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary: The `TestCPUValue` test case always fails when execute in local machine without `InContainer() == true`.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

File: util/cpu/cpu_test.go [hunk 1]
```diff
@@ -23,11 +23,15 @@ import (
...
 	"github.com/pingcap/tidb/resourcemanager/scheduler"
 	"github.com/pingcap/tidb/resourcemanager/util"
+	"github.com/pingcap/tidb/util/cgroup"
 	"github.com/pingcap/tidb/util/cpu"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCPUValue(t *testing.T) {
+	if !cgroup.InContainer() {
+		t.Skip()
```
Comment (hi-rustin) [inline util/cpu/cpu_test.go:33]: Could you please add a comment here?

Comment (hi-rustin) [reply util/cpu/cpu_test.go:33]: Or we can set a message for this skip, like t.Skip("xxx").

Comment (Defined2014) [reply util/cpu/cpu_test.go]: Done

**Review by hi-rustin — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**